src/main/java/hu/rxd/toolbox/qtest/diff/classifiers/TextMatchClassifier.java

```diff
@@ -0,0 +1,38 @@
+package hu.rxd.toolbox.qtest.diff.classifiers;
+
+import java.util.Iterator;
+
+import hu.rxd.toolbox.qtest.diff.DiffClassificator.Classifier;
+import hu.rxd.toolbox.qtest.diff.DiffClassificator.DiffObject;
+
+public class TextMatchClassifier implements Classifier {
+  
+  private String text;
+
+  public TextMatchClassifier(String text) {
+    this.text = text;
+  }
+  
+  @Override
+  public String getName() {
+    return String.format("textMatch_%s", normalize(text));
+  }
+
+  private String normalize(String textToNormalize) {
+    return textToNormalize.toLowerCase().replaceAll("\\s+","_");
+  }
+
+  @Override
+  public boolean accept(DiffObject dio) {
+    Iterator<String> rIter = dio.r.iterator();
+
+    while (rIter.hasNext()) {
+      String rLine = rIter.next();
+
+      if (!rLine.contains(text)) {
+        return true;
```

Comment:
this matcher seems to be a bit odd to me...

* it doesn't check anything on the left side
* it checks that the changes in R has at least 1 line which doesn't contain the predefined pattern

I think a testcase would help